fhirpipe/api/API.md

```diff
@@ -0,0 +1,60 @@
+# FHIRPIPE API
+
+
+## Run
+
+Runs the ETL with on resources for which the ids are given and loads the created objects in the Mongo database.
+
+### Request
+
+`POST /run`
+
+### Body parameters
+
+The route expects the following parameters, taking these ones as default is not provided.
+
+```
+{
+    "mapping": None,
+    "resource_ids": None,
+    "override": False,
+    "chunksize": None,
+    "bypass_validation": False,
+    "skip_ref_binding": False,
+    "multiprocessing": False,
+    "credentialId"
+}
+```
+`credentialId` has no default value and is required.
+
+These are the arguments taken by the `run()` function of fhirpipe.
+
+### Response
+
+This route only return a success response if no error occurred in the process.
+
+### Errors
+
+Throws an `OperationOutcome` error with the error message if any error occurred while running fhirpipe or if credentials where not provided for the source DB in Pyrog. 
```

Comment:
can you describe what the `OperationOutcome` object looks like in the  HTTP response ?